packages/eslint-checker/README.md

```diff
@@ -0,0 +1,51 @@
+# `eslint-checker`
+
+An ESLint type checker plugin for [StrykerJS](https://stryker-mutator.io), the JavaScript Mutation testing framework.
+This plugin enables lint testing on mutants, so you won't have to waste time on mutants which results in a lint error.
+
+# Features
+
+👽 Lint check each mutant. Invalid mutants will be marked as CompileError in your Stryker report.
+
+🧒 Easy to setup, only your `.eslintrc.*` file is needed.
+
+🔢 Lint check is done in-memory, no side effects on disk.
+
+🎁 Support for javascript projects (including type-aware projects), as well as typescript projects.
+
+# Configuring
+
+You can configure the typescript checker in the `stryker.conf.js` (or `stryker.conf.json`) file.
+
+```json
+{
+  "checkers": ["eslint"],
+  "lintConfigFile": ".eslintrc.json"
+}
+```
+
+## `lintConfigFile` [`string`]
+
+Default: `.eslintrc.*`
+
+The path to your [eslint configuration file](https://eslint.org/docs/user-guide/configuring/configuration-files#using-configuration-files).
+
+_Note: the following eslint options are always overridden by @stryker-mutator/eslint-checker to avoid false positives, based on a similar reasoning to that in [@stryker-mutator/typescript-checker](https://stryker-mutator.io/docs/stryker-js/typescript-checker/)_
+
+```json
+{
```

Comment:
I think it would also be smart to disable https://eslint.org/docs/rules/max-len due to the fact that mutation switching will result in (much) longer lines of code

Reply:
Yes, I was thinking of getting a minimal set of rules in initially, and refining this later. Happy to add any rules you think are sensible before merging this PR